src/test/java/bio/terra/service/rawls/WorkspaceDetailsTest.java

```diff
@@ -0,0 +1,30 @@
+package bio.terra.service.rawls;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import bio.terra.common.category.Unit;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.UUID;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag(Unit.TAG)
+class WorkspaceDetailsTest {
+  private static final String WORKSPACE_ID = UUID.randomUUID().toString();
+  private static final String NAMESPACE = "namespace";
+  private static final String NAME = "name";
+
+  @Test
+  void workspaceDetails() throws JsonProcessingException {
+    ObjectMapper objectMapper = new ObjectMapper();
+    String content =
+        String.format(
+            "{\"workspaceId\": \"%s\", \"namespace\": \"%s\", \"name\": \"%s\", \"unknown\": \"property\"}",
+            WORKSPACE_ID, NAMESPACE, NAME);
```

Comment:
It might be a little more legible to use a text block with `formatted` for this string.
```suggestion
    String content = """
        {"workspaceId": "%s", "namespace": "%s", "name": "%s", "unknown": "property"}"""
        .formatted(WORKSPACE_ID, NAMESPACE, NAME);
```

Reply:
I agree, thanks for suggesting!